examples/01_example_wsiread.ipynb

```diff
@@ -21,20 +21,40 @@
     "id": "p3l08vukKAWo"
```

Comment:
<p> </p><p> </p><p><u>Check that "colab" appears in the browser address bar after you click on "Open in Colab".</u></p><p>Check that "colab" or "kaggle", as appropriate, appear in the address bar.</p><p> </p><p><u>on either Github or colab</u></p><p>on Github, Colab or Kaggle</p><p> </p><p><u>locally or  outside Colab</u></p><p>on any platform, except for Colab or Kaggle,</p><p> </p><p><em>Remarks: 1) The user may have access to machines that run notebooks remotely, so "locally" is misleading. 2) Advice to use pip with --ignore is only available in installation.rst. It should appear in README.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9